node/tests/TestUtilities.ts

```diff
@@ -4,6 +4,7 @@
 
 import { beforeAll, expect } from "@jest/globals";
 import { exec } from "child_process";
+import parseArgs from "minimist";
```

Comment:
Please, update developer.md and/or package.json to include this module

Reply:
It's updated, look at the pr

Reply:
I see, need to update dev doc.